### ISSUE TITLE
Request to support episode thumbnails on WDTV Live Streaming Player

### ISSUE THREAD

**Issue by psouza4** [psouza4/mediacentermaster#1379]
Originally reported on Google Code with ID 1382

```
Describe the new feature or change:

Request to add an option to generate episode thumbnails in the right format for the
WDTV Live Streaming Player.

I think the problem is that currently, thumbnails are:
- named after the episode number (e.g. 2803651.jpg)
- have the jpg extension
- are put in a separate folder called metadata

The thumbnails I have working on the WDTV Live Streaming Player follow this example:

- show\S03E21.mp4
- show\S03E21.xml
- show\S03E21.metathumb [where this file is a jpg given the metathumb extension]

So in other words, the thumbnails need to be in the same folder as the video file,
and they need to have the 'metathumb' extension. They also need to have the same name
as the video file.

I hope you can find time to add this feature. I know a lot of people use the WDTV Live
device and it would be awesome if you could get thumbs working.

Thanks!

List any references (websites or forum threads) that help describe this
request:

- http://forums.mediacentermaster.com/viewtopic.php?f=4&t=7011
```

Reported by `neil@wheatleycopywriting.co.uk` on 2013-05-24 07:11:08
